src/Orbit.ts

```diff
@@ -0,0 +1,49 @@
+import { Schema } from '@orbit/data'
+import Store from '@orbit/store'
+import JSONAPISource from '@orbit/jsonapi'
+import Coordinator, { RequestStrategy, SyncStrategy, EventLoggingStrategy, LogLevel } from '@orbit/coordinator'
+
+export default class Orbit {
+  public currentSchema: any
```

Comment:
we may want to sent `readonly` on some of these, or use getters

Reply:
@tniezg I will be preparing interfaces for each variable. It has `readonly` method inside https://basarat.gitbooks.io/typescript/docs/types/readonly.html